### ISSUE TITLE
Parent poll failed

### ISSUE THREAD

**Issue by MatjazGo** [spyder-ide/spyder#8288]
### What steps will reproduce the problem?

Just freshly installed 64 bit Anaconda 5.3.1 on Windows 7 64 bit and I got this error on my first run of Spyder (Python 3.7):

Python 3.7.0 (default, Jun 28 2018, 08:04:48) [MSC v.1912 64 bit (AMD64)]
Type "copyright", "credits" or "licence" for more information.

IPython 6.5.0 -- An enhanced Interactive Python.
C\Users\...\Anaconda3\lib\site-packages\ipykernel\parentpoller.py:116: UserWarning: Parent poll failed. If the frontend dies,
    the kernal may be left running. Please let us know
    about your system (bitness, Python, etc.) at
    ipython-dev@scipy.org
  ipython-dev@scipy.org""")

### Traceback
```python-traceback

WARNING:spyder.widgets.github.backend:failed to send bug report on github. response={'code': 403, 'json': {'message': 'At least one email address must be verified to do that.', 'documentation_url': 'https://help.github.com/articles/adding-an-email-address-to-your-github-account'}}
WARNING:spyder.widgets.github.backend:failed to send bug report on github. response={'code': 403, 'json': {'message': 'At least one email address must be verified to do that.', 'documentation_url': 'https://help.github.com/articles/adding-an-email-address-to-your-github-account'}}
```

## Versions

* Spyder version: 3.3.1 
* Python version: 3.7.0
* Qt version: 5.9.6
* PyQt5 version: 5.9.2
* Operating System: Windows 7

### Dependencies

```
pyflakes >=0.6.0  :  2.0.0 (OK)
pycodestyle >=2.3 :  2.4.0 (OK)
pygments >=2.0    :  2.2.0 (OK)
sphinx >=0.6.6    :  1.7.9 (OK)
rope >=0.9.4      :  0.11.0 (OK)
jedi >=0.9.0      :  0.12.1 (OK)
nbconvert >=4.0   :  5.4.0 (OK)
pandas >=0.13.1   :  0.23.4 (OK)
numpy >=1.7       :  1.15.1 (OK)
sympy >=0.7.3     :  1.1.1 (OK)
cython >=0.21     :  0.28.5 (OK)
qtconsole >=4.2.0 :  4.4.1 (OK)
IPython >=4.0     :  6.5.0 (OK)
matplotlib >=2.0.0:  2.2.3 (OK)
pylint >=0.25     :  2.1.1 (OK)
```

**Comment by ccordoba12** [comment]
Thanks for reporting. This is a known problem with Python 3.7 on Windows and we don't know what's causing it and how to fix it, sorry.

Our recommendation is to use Python 3.6 for now.

**Comment by CAM-Gerlach** [comment]
@ccordoba12 I can repro this in QtConsole on Py3.7/Win 8.1, so the problem isn't with Spyder's codebase. Should I report it there and see if any of the Jupyter guys know how to fix it, or what the cause is?

**Comment by ccordoba12** [comment]
We're talking about this on issue #8205.

**Comment by sureshd2014** [comment]
The same problem occurs with Python 3.6 also.
"site-packages\ipykernel\parentpoller.py:116: UserWarning: Parent poll failed."

**Comment by sureshd2014** [comment]
After that event of Kernel dying, I restart the kernel and it happens again/ I have to close the Spyder IDE and open it again and for a while the problem does not occur.

**Comment by CAM-Gerlach** [comment]
@sureshd2014 We appreciate you sharing the report, but again, please do so on #8205 , since this is a duplicate of that issue. Also, provide your ``conda list`` output inside a ``<details></details>`` block. Thanks!

**Comment by ccordoba12** [comment]
I don't think this is a duplicate because that warning can't cause a kernel failure. @sureshd2014's problem is most probably caused by an error in his/her code.

**Comment by CAM-Gerlach** [comment]
Oh, I missed their second post implying that their kernel actually died. The issue reported in the OP here though is a duplicate (as you mentioned).

**Comment by sureshd2014** [comment]
Please note that this problem has been solved in Spyder 3.3.2.
I upgraded the Spyder on my system yesterday and in 24 hours of continuous
running of Motion Detection script, the Kernal has not crashed once.

Looks like a winner.

- Suresh

**Comment by sureshd2014** [comment]
I walked through my code and dud unit tests and checked he memory usage in
Windows 10 Task Manager. Couldn't find anything out of the ordinary.

Then I upgraded my Spyder to version 3.3.2. and have been running the same
script for over 24 hours continuously.  The problem has not occurred.

Looks like it has been fixed (or worked around) in Spyder version 3.3.2.

Thanks and Best Wishes
Suresh

**Comment by CAM-Gerlach** [comment]
Great @sureshd2014 ! Glad to hear.
